### ISSUE TITLE
Database dependency in for tomcat playbook

### ISSUE THREAD

**Issue by Natkeeran** [Islandora/documentation#1282]
I am wondering if the following is a documentation to ansible ticket.

Recent [changes](https://github.com/Islandora-Devops/claw-playbook/commit/1a7754450ea8521ae0b117a5858b4cb17984cc81) necessitate that database be installed for tomcat playbook.  

If you are installing tomcat into a separate server, then [database.yml](https://github.com/Islandora-Devops/claw-playbook/blob/dev/database.yml) or the roles within it would need to be part of the tomcat.yml.  This will install the required database in the tomcat server.  

We may want to consider a situation where the databases are served from another server as well (i.e we need to configure to enable to connect externally.)

**Comment by seth-shaw-unlv** [comment]
This gets at a larger restructuring question. Tomcat doesn't need a database, but Fedora does. However, our playbook assumes that the Tomcat hosts will house Fedora and it's database. 

In our local production setup we split the tomcat.yml into both a cantaloupe.yml and fedora.yml and redefined our hosts so that the host getting Drupal and Cantaloupe _don't_ get the database role (because our Drupal is pointing to an existing database server) while the Fedora host _does_ (because we wanted to test the scale of that database on the Fedora machine before making the decision to move it elsewhere).

So, long story short, we really should be using the inventory to get the database on the right hosts rather than adding the database role to the tomcat playbook.
